README.md

```diff
@@ -122,6 +122,9 @@ Mypy can also be integrated into [Flake8] using [flake8-mypy].
 [Flake8]: http://flake8.pycqa.org/
 [flake8-mypy]: https://github.com/ambv/flake8-mypy
 
+Mypy can also be set up as a pre-commit hook:
+
+* [pre-commit mirrors-mypy](https://github.com/pre-commit/mirrors-mypy)
```

Comment:
But it is still a single element list, why not just add another normal sentence right after one about flake8?